src/core/biblio-db.js

```diff
@@ -78,7 +40,7 @@ export const biblioDB = {
     if (await this.isAlias(id)) {
       id = await this.resolveAlias(id);
     }
-    return this.get("reference", id);
+    return await this.get("reference", id);
```

Comment:
```suggestion
    return this.get("reference", id);
```

Reply:
I'd rather say "if you await something in a function, rather await all", because returning a raw promise sometimes causes a tricky situation:

```js
try {
  const result = await myPromise;
  return result.doAsync(); // A rejected promise here won't call the catch block!
} catch {
  console.error("Ouch!");
}
```

Reply:
In general,  I agree. My general preference is usually to either return the raw promise in cases where it's a simple return type (e.g., boolean here)... or always put the result into a variable, as it makes debugging much easier, as one can check the `result` before returning it. 

@Swapnilr1, we leave it to you to decide what is best.